administration/bundles.md

```diff
@@ -40,9 +40,26 @@ openhab> bundle:list
 
 The _start_ command works accordingly:
 
-```
+```python
 openhab> bundle:start 209
 openhab> bundle:list
 ...
 209 | Active   |  80 | 2.0.0.b3              | Network Binding
 ```
+
+## Naming Convention For Bundles
+
+Bundles are named according to the following convention.
+
+```java
+<prefix>-<type>-<id>
+```
+
+where
+
+- **prefix** is ALWAYS "openhab" 
+- **type** is the add-on type, e.g. "binding" or "ui"
+- **id** is the identifier for this bundle
+  (It is also the id that you would have to use openHAB 1.x in addons.cfg, e.g. "hue", "map", etc.)
+
+  This is used for example to assign them the correct features for the karaf console or to sort them in different tabs within the Paper UI.
```

Comment:
Consider, "The bundle identifier may be useful to assign the correct features in the karaf console, or to sort the bundle into different tabs within the PaperUI."

If this is not what is meant,  consider revising the text.  The example as written is not clear.

Reply:
Hm.

The intention was to give at least some examples why the naming convention is existing.
I think automatic sorting in Paper UI is easier to grasp than the Karaf thing.
(Especially for beginners, who are browsing through the docs.)

Maybe i should change it to one Paper UI example and clarify that there are many other usecases.

Wdyt?

Reply:
Yep - one solid example is enough, and the one about sorting stuff in a UI is easily understood.  Beginners may not yet even be aware... "What is this karaf console of which you speak?".  Boom - gone for an hour of Google searching, only to figure out that, in the early stages, they don't need to know anything about it.